connect/api.go

```diff
@@ -57,3 +58,46 @@ func GetProduct(productQuery Product) (Product, error) {
 	}
 	return remoteProduct, nil
 }
+
+// updateSystemAPI resend the system's hardware info to SCC.
+// https://github.com/SUSE/connect/wiki/SCC-API-%28Implemented%29#update-system
+func updateSystemAPI(distroTarget, namespace string, instanceData []byte) error {
+	payload, err := updatePayload(distroTarget, namespace, instanceData)
+	if err != nil {
+		return err
+	}
+	body := bytes.NewReader(payload)
```

Comment:
I think it would be more natural and convenient for callHTTP() to receive a []byte instead of io.Reader for the body. I will propose a change for that

Reply:
done

Reply:
nice. thx!